plot/plot.app.js

```diff
@@ -412,8 +410,14 @@
                         defer.resolve(plot_values);// TODO: figure out how to return an error to catch clause without breaking code
                         return defer.promise;
                     }
-
-                    server.http.get(uri).then(function(response) {
+                    var headers = {};
+                    headers[ERMrest.contextHeaderName]=ConfigUtils.getContextHeaderParams();
+                    if(UriUtils.getQueryParams($window.location.href).pcid)
+                        headers[ERMrest.contextHeaderName].pcid=UriUtils.getQueryParams($window.location.href).pcid;
+                    if(UriUtils.getQueryParams($window.location.href).ppid)
+                        headers[ERMrest.contextHeaderName].ppid=UriUtils.getQueryParams($window.location.href).ppid;
+
+                    server.http.get(uri,{ headers: headers}).then(function(response) {
```

Comment:
Like in treeview, we want to call `ERMrest._certifyContextHeader()` before this get request. This makes sure it is a properly formed header for our logging purposes.

Reply:
Made required changes